src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let totalValue = days * 40;
+
+  if (days >= 3 && days < 7) {
+    totalValue = days * 40 - 20;
+  } else if (days >= 7) {
+    totalValue = days * 40 - 50;
```

Comment:
Replace the hardcoded value '50' with a named constant to avoid magic numbers in the code.